video/out/wayland_common.h

```diff
@@ -131,6 +133,9 @@ struct vo_wayland_state {
     bool                    cursor_visible;
     int                     allocated_cursor_scale;
     uint32_t                pointer_id;
+
+    /* Screen Saver */
+    struct vo_wayland_screensaver *screensaver;
```

Comment:
and what if I don't have `HAVE_GIO`? Adding more state to the struct is one thing but I wouldn't even have the `vo_wayland_screensaver` struct definition.